### DISCUSSION TITLE
Example beam_roll_up in StructuralMechanicsApplication

### DISCUSSION THREAD

**Discussion by mnabideltares** [KratosMultiphysics/Kratos#10949] [General]
We are trying to reproduce the example in:

https://github.com/KratosMultiphysics/Examples/tree/master/structural_mechanics/validation/beam_roll_up

However Kratos generates the following error.

            from KratosMultiphysics.ExternalSolversApplication  import *
            ModuleNotFoundError: No module named 'KratosMultiphysics.ExternalSolversApplication'

We are not able to find ExternalSolversApplication. Is this application available?

Thank you and best regards

**Comment by loumalouomega** [comment]
No, now is called LinearSolverApplication, sorry for the nuisance

**Reply by mnabideltares** [reply]
@loumalouomega Thank you for the fast reply, and clear answer.
